### PR TITLE
#3560 Logs not being saved

### LINKED ISSUE #3560: Downloaded logs are not being integrated
## Describe the bug

Logs being downloaded are not being saved into realm. I have been waiting for 30 minutes and no logs have downloaded!

### To reproduce

1. Go to Add a sensor
2. Think the first lot of logs will download
3. wait awhile
3. no more will be saved after that

### Expected behaviour

Logs should be saved

### Proposed Solution

Thinks its to do with this: https://github.com/openmsupply/mobile/blob/3cfacf047f7e99cd3b58ec17136520eed16d622e/src/actions/Bluetooth/SensorDownloadActions.js#L107-L114

The `nextPossibleLogTime` being passed into `calculateNumberToSave` should be a unix timestamp?

### Version and device info

- App version: 7.0.0-rc4
- Tablet model: N/A
- OS version: N/A

### Additional context

N/A

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #3560

## Change summary

- Added hopefully the correct next log timestamp
- ran tests and noticd sensormanager tests failing

## Testing

- [ ] After downloading logs they're saved

### Related areas to think about

Be good to test this using a physical

### CODE REVIEW

**Comment by joshxg** [comment]
Argh sorry this actually will allow for integrating records but now their timestamps are incorrect. So don't review yet

**Comment by joshxg** [comment]
@mark-prins @katherine-sussol sorry - this should be ready now.

Also should fix #3561

File: src/actions/Bluetooth/SensorDownloadActions.js [hunk 1]
```diff
@@ -98,26 +95,31 @@ const downloadLogsFromSensor = sensor => async dispatch => {
...
+            // must be in the past.
+            const downloadBoundary = moment
+              .max([
+                moment(programmedDate),
+                moment(logDelay),
+                moment(mostRecentLogTime).add(logInterval, 's'),
+              ])
+              .unix();
+
+            const nextTimestamp = mostRecentLog ? moment.max([mostRecentLogTime, logDelay]) : null;
```
Comment (katherine-sussol) [inline src/actions/Bluetooth/SensorDownloadActions.js:111]: Do `mostRecentLogTime` and `logDelay` need to be moment-ified here? Added a new sensor and been getting problems with getting the logs to load as is 🙀

Comment (joshxg) [reply src/actions/Bluetooth/SensorDownloadActions.js]: Ah thanks, sure does: a61a50e

Buuut.. I also just realised that it was quite right, anyway? So I tried to fix it with: 
539e903

Comment (joshxg) [reply src/actions/Bluetooth/SensorDownloadActions.js]: aaaand officially a mess f9d226d

**Review by katherine-sussol — APPROVED** [review]
👍  This LGTM now - just verified on a real device and new ones appear to be getting saved now.

**Pull request merged**